src/AppMenuButtonGroup.cc

```
@@ -593,6 +592,10 @@ void AppMenuButtonGroup::updateAppMenuModel()
             if (AppMenuButton *b = getAppMenuButton(m_currentIndex)) {
                 b->setChecked(true);
             }
+
+            if (wasSearchOpen && !savedQuery.isEmpty()) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)